app/src/main/java/org/schabi/newpipe/settings/AppearanceSettingsFragment.java

```diff
@@ -51,16 +46,11 @@ public void onCreatePreferences(final Bundle savedInstanceState, final String ro
         } else {
             removePreference(nightThemeKey);
         }
-
-        captionSettingsKey = getString(R.string.caption_settings_key);
-        if (!CAPTIONING_SETTINGS_ACCESSIBLE) {
-            removePreference(captionSettingsKey);
```

Comment:
Where is this preference removed now?

Reply:
Ty for the review 😄 
It's never removed because we are always running above Android 4.4